scripts/check_compliance.py

```diff
@@ -1295,8 +1300,8 @@ def _main(args):
 
         test = testcase()
         try:
-            print("Running {:16} tests in {} ..."
-                  .format(test.name, test.path_hint))
+            print(f"Running {test.name:16} tests in "
+                  f"{GIT_TOP if test.path_hint == '<repo>' else test.path_hint} ...")
```

Comment:
I don't get this. If GIT_TOP was successfully found, why was `path_hint` not already initialized?

Reply:
`path_hint` is a class variable, so it gets initialized when the class is defined (when the Python interpreter runs the `class ...` statement).

This is the only place where `path_hint` is used, so rewriting it from `"<git-top>"` (new name) to `GIT_TOP` seemed unnecessary. Nicer to have it stateless I think.

Reply:
Sorry for the previous comment, it assumed you _were_ rewriting path_hint.

As I wrote in the main thread, I think it would be better to rename `path_hint` to `directory_tested` or something like it and then use that to unhardcode `cwd` in `subprocess.check_output(cwd=directory_tested,... )` in `GitLint`,`CheckPatch` and maybe others I missed.

This would be added a couple lines above:
```python
if testcase.directory_tested == "<git-top>":
      testcase.directory_tested = GIT_TOP
```